tokio/Cargo.toml

```diff
@@ -28,6 +28,7 @@ default = [
   "blocking",
   "fs",
   "io-util",
+  "local",
```

Comment:
I would avoid the feature flag for now... just include it with `rt-full` or something. I would guess we will add a `task-util` feature flag before final. There is more feature flag work to do.

Reply:
👍 I'll remove the separate flag, I think it only made sense when `local` was a module at top level, anyway. I do think it should be available with just the current-thread scheduler, though, esp. since the compat crate will use it